server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1586,6 +1586,11 @@ public void update(LiveTServerSet current, Set<TServerInstance> deleted,
       cleanListByHostAndPort(serversToShutdown, deleted, added);
     }
 
```

Comment:
I am wondering if I can simply remove all of the removes above this line now since doing a retainAll of the current servers theoretically would take care of those.

Reply:
Upon further examination, I believe I may be able to only remove line 1579.  However it is certainly not hurting anything.  I will leave this as is for now.

Reply:
I was wondering the same thing.  It is a bit of a mess.